src/components/ImportHistoryPage.vue

```diff
@@ -0,0 +1,98 @@
+<template>
```

Comment:
I'd prefer having this as a modal instead of being its own page since the page only contains three buttons in total.

Reply:
The only reason I added it as a second page is because I felt like there was too much code that doesn't really have much to do with history, it's more about importing. Also, if we're going to support importing from other platforms in the future that will be even more code and possibly instructions, similar to the subscription import page

Reply:
I get your point with the code structure, however that wouldn't be an issue if you just keep everything in that file and just wrap it into a `ModalComponent`. And I don't think there is that much of additional information needed as on the subscriptions page since not many other projects let you export the history currently, so instructions for others shouldn't be needed as much as on the subscriptions import page.

Reply:
You were right, it's much cleaner this way.